src/ui/setup/dynamic/devtools.ts

```diff
@@ -56,10 +56,12 @@ declare global {
 }
 
 const url = new URL(window.location.href);
-const recordingId = url.searchParams.get("id")!;
+const recordingId = url.pathname.match(/\/([a-z0-9-]+)/i)?.pop();
```

Comment:
Seems like we should explicitly do `/recording/:uuid` since we would well also want `/workspace/:uuid` or something at some point. This regex also needs to start with `^`.

Reply:
That's a good point. I hadn't considered teams as an endpoint but agree we should plan for it. That regex will hopefully go away and was just a quick hack to get the value for testing things out.